generator yo nodeserver is broken

Looks like the update yesterday has broken https://github.com/ibm-developer/generator-nodeserver

We get the following trying to run any node generation

Error: Cannot find module '../../lib/handlebars.js'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/generator-nodeserver/node_modules/generator-ibm-web/generators/app/index.js:19:20)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.get [as ibm-web:app] (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/store.js:39:27)
    at Store.get (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/store.js:62:39)
    at Environment.get (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/environment.js:342:23)
    at Environment.getByPath (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/environment.js:359:19)

    at ChildProcess.exithandler (child_process.js:276:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:915:16)
    at Socket.stream.socket.on (internal/child_process.js:336:11)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at Pipe._handle.close [as _onclose] (net.js:561:12)
  killed: false,
  code: 1,
  signal: null,

--- a/package.json
+++ b/package.json
@@ -5,7 +5,8 @@
   "license": "Apache-2.0",
   "main": "index.js",
   "files": [
-    "generators"
+    "generators",
+    "lib"
   ],
   "keywords": [
     "yeoman-generator"

